.gitignore

```diff
@@ -0,0 +1,15 @@
+ssets-cache/
+webassets-external/
+.DS_Store
+*.pyc
+*.pyo
+env
+venv
+.env
```

Comment:
Can we add *.env to cover the config.env file?

Reply:
+1 got it